src/protocols/rdp/keymaps/ja_jp_qwerty.keymap

```diff
@@ -20,6 +20,9 @@
 parent  "base"
 name    "ja-jp-qwerty"
 freerdp "KBD_JAPANESE"
+freerdp_keyboard_type "KBD_TYPE_JAPANESE"
```

Comment:
Is this something that should be set for the other layouts?

Reply:
FreeRDP has default values.
https://github.com/FreeRDP/FreeRDP/blob/master/libfreerdp/core/settings.c#L345-L347
So, no need to specify if there is no problem for other layouts.